frontend/src/app/voice-ai/components/audio-pulse/AudioPulse.tsx

```
@@ -0,0 +1,48 @@
+export default function AudioPulse({ active, volume, hover }: AudioPulseProps) {
+  const lines = useRef<HTMLDivElement[]>([]);
+
+  useEffect(() => {
+    let timeout: number | null = null;
+    const update = () => {
+      lines.current.forEach(
+        (line, i) =>
+          (line.style.height = `${Math.min(
+            24,
+            4 + volume * (i === 1 ? 400 : 60)
+          )}px`)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)